Feature: use condition in when expression (#631)
codemagic-ci-cd/codemagic-docs: content/yaml/yaml-getting-started.md

        fi
```

{{<notebox>}}
Read more about configuring [additional conditions to run or skip](/yaml/yaml-conditional-run) builds or build steps.

> {{</notebox>}}

Reviewer: let's move this note upwards to the triggering section and add it after the note about using API to trigger builds

> Visit [conditional run and skip](/yaml/yaml-conditional-run) page to set up `when` section with `condition` and `changeset`. Conditional run ans skip applies for builds and specific build steps.

Reviewer: ```suggestion
Read more about configuring [additional conditions to run or skip](/yaml/yaml-conditional-run) builds or build steps.
```
